### PR TITLE
Add BD - Advanced Weapons Pack

### PR DESCRIPTION
**Pull request by politas**
Replaces #2870

### CODE REVIEW

File: NetKAN/BD-AWP.netkan [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "spec_version": "v1.4",
+    "license": "MIT",
+    "identifier": "BD-AWP",
+    "install": [
+        {
+            "find": "AWP",
+            "install_to": "GameData"
+        },
+        {
+            "file": "INSTALLATION GUIDE.txt",
```
Comment (plague006) [inline NetKAN/BD-AWP.netkan]: We can safely exclude the installation of this file. Having it in the install stanza leaves it vulnerable to breaking if the author moves it or renames it.

**Comment by plague006** [comment]
Part of me wants to hardcode an abstract for our users, but I think the mod's name is descriptive enough to make up for its deficiencies.

**Comment by politas** [comment]
At least it has something. Some of the KerbalStuffBot submissions are just awful.

**Comment by plague006** [comment]
Very true.

**Pull request merged**